docs/en/week07/07-3.md

```diff
@@ -0,0 +1,116 @@
+# Application of Autoencoders
+## DALL-E: Creating Images from Text
+
+A neural network that creates images from text captions. 
+![](https://i.imgur.com/FosonFM.jpg)
```

Comment:
Where's the caption?

Reply:
Text caption? It is in the image.

Reply:
The figure's caption is missing.
**Fig 1:** description of the figure 1

Also, please, read [the instructions](https://atcold.github.io/pytorch-Deep-Learning/en/about/)!